cli/cmd/plugin/ui/web/tanzu-ui/src/shared/components/DeployProgress/DeployProgress.tsx

```diff
@@ -68,29 +72,50 @@ function DeployProgress() {
         if (logData && logData.type === LogTypes.LOG) {
             const logLine = formatLog(logData.data);
             setLogMessageHistory((prev) => prev.concat([logLine]));
-        } else if (logData && logData.type === LogTypes.STATUS) {
-            setStatusMessageHistory((prev) => logData.data);
+        } else if (logData && logData.type === LogTypes.PROGRESS) {
+            handleDeploymentProgress(logData.data);
         }
     }, [websocketSvc.lastMessage]);
 
     /**
-     * @method formatLog
-     * @param log - individual log object
+     * Handler for storing last message indicating deployment progress.
+     * Displays page notification once cluster creation success or failure is reached.
+     */
+    function handleDeploymentProgress(statusData: StatusMessageData) {
+        const latestStatusMsg: StatusMessageData = statusData;
+
+        if (statusData.status === DeploymentStates.SUCCESSFUL) {
+            setNotification({
```

Comment:
I'm pondering how other pages might be able to access this notification (if that's what we want). Should it be put in the App store? Should the closeCallback clear it from there?